src/Uno.UI/UI/Xaml/Controls/Layouter/Layouter.cs

```diff
@@ -140,9 +139,17 @@ public Size Measure(Size availableSize)
 			}
 		}
 
+		[Pure]
+		private static bool IsCloseReal(double a, double b)
+		{
+			var x = Abs((a - b) / (b == 0d ? 1d : b));
```

Comment:
1. This is called pretty often no? Might be worth rewriting to avoid the division

2. There seems to be an identical method here for WASM: https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/FrameworkElement.Layout.wasm.cs#L138 - should also be written, or better yet remove the duplication